src/request.rs

```diff
@@ -37,18 +37,20 @@ impl<'a> Request<'a> {
 
     /// Calls the method using the given `Client`.
     ///
-    /// This will send the request to the `/` URL.
-    ///
     /// Returns a `RequestResult` indicating whether the request was sent and processed successfully
     /// (according to the rules of XML-RPC).
-    pub fn call(self, client: Client) -> RequestResult {
+    pub fn call(self, client: Client, url: &str) -> RequestResult {
         // First, build the body XML
         let mut body = Vec::new();
         try!(self.write_as_xml(&mut body));
 
+        use hyper::header::ContentType;
+        use hyper::mime::{Mime, TopLevel, SubLevel, Attr, Value};
```

Comment:
You can move these to the top of the file. I like having all imports at the top in most cases.